source/MapPlanetCard.cpp

```diff
@@ -44,10 +44,23 @@ MapPlanetCard::MapPlanetCard(const StellarObject &object, unsigned number, bool
 	hasShipyard = planet->HasShipyard();
 	hasOutfitter = planet->HasOutfitter();
 
-	reputationLabel = !hasSpaceport ? "No Spaceport" :
-		GameData::GetPolitics().HasDominated(planet) ? "Dominated" :
-		planet->GetGovernment()->IsEnemy() ? "Hostile" :
-		planet->CanLand() ? "Friendly" : "Restricted";
+	if(!hasSpaceport)
+		reputationLabel = "No Spaceport";
+	else
+	{
+		Planet::Friendliness friendliness = planet->GetFriendliness();
+		switch(friendliness)
+		{
+			case Planet::Friendliness::FRIENDLY:
+				reputationLabel = "Friendly";
```

Comment:
You need breaks here, or else the cases will all fallthrough.

Reply:
oops